CMakeLists.txt

```diff
@@ -293,8 +293,11 @@ add_library(avif_enable_warnings INTERFACE)
 if(MSVC)
     message(STATUS "libavif: Enabling warnings for MSVC")
     target_compile_options(
-        avif_enable_warnings INTERFACE /W4 # For clang-cl, /W4 enables -Wall and -Wextra
-                                       /wd4324 # Disable: structure was padded due to alignment specifier
+        avif_enable_warnings
+        INTERFACE /W4 # For clang-cl, /W4 enables -Wall and -Wextra
+                  /wd4232 # Disable: address of dllimport 'dllimport' is not static,
```

Comment:
Though unlikely it sounds like this would be an error if `/Za` was used. Though I guess the alternative would be to manually load the functions.

Reply:
Or have a wrapper as you suggested with the totw reference in the bug.

Reply:
Thanks for taking a look at the MSVC C4232 warning documentation. The wrapper would also fix the CFI issue, so it seems like the best long-term solution.